slow_conv3d: Use at::sum for grad_bias accumulation

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #66165
* #65759
* __->__ #65758
* #65757
* #65737
* #65726

The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

Differential Revision: [D31257872](https://our.internmc.facebook.com/intern/diff/D31257872)

peterbell10 committed:
slow_conv3d: Use at::sum for grad_bias accumulation

The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

[ghstack-poisoned]

peterbell10 committed:
Update on "slow_conv3d: Use at::sum for grad_bias accumulation"


The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

[ghstack-poisoned]

ngimel commented:
@ngimel has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31257872).

peterbell10 committed:
Update on "slow_conv3d: Use at::sum for grad_bias accumulation"


The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

Differential Revision: [D31257872](https://our.internmc.facebook.com/intern/diff/D31257872)

[ghstack-poisoned]

ngimel commented:
@ngimel has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31257872).

peterbell10 committed:
Update on "slow_conv3d: Use at::sum for grad_bias accumulation"


The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

Differential Revision: [D31257872](https://our.internmc.facebook.com/intern/diff/D31257872)

[ghstack-poisoned]

peterbell10 committed:
Update on "slow_conv3d: Use at::sum for grad_bias accumulation"


The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

Differential Revision: [D31257872](https://our.internmc.facebook.com/intern/diff/D31257872)

[ghstack-poisoned]

peterbell10 committed:
Update on "slow_conv3d: Use at::sum for grad_bias accumulation"


The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

Differential Revision: [D31257872](https://our.internmc.facebook.com/intern/diff/D31257872)

[ghstack-poisoned]

peterbell10 committed:
Update on "slow_conv3d: Use at::sum for grad_bias accumulation"


The same change has been made in conv2d, the proper algorithm is both
faster and gives more precision.

Differential Revision: [D31257872](https://our.internmc.facebook.com/intern/diff/D31257872)

[ghstack-poisoned]

ngimel commented:
@ngimel has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D31257872).
